File: Data Structures & Algorithms/majority-element/submission-5.java
class Solution {
    public int majorityElement(int[] nums) {
        Map<Integer, Integer> map = new HashMap<>();
        int maxCount = 0;
        int majority = nums[0];

        for (int i = 0; i < nums.length; i++){
            if (!map.containsKey(nums[i])){
                map.put(nums[i], 1);
            }
            else{
                int x = map.get(nums[i]);
                x++;
                map.put(nums[i], x);
            }

            if (map.get(nums[i]) > maxCount){
                maxCount = map.get(nums[i]);
                majority = nums[i];
            }
        }
        return majority;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-5.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        h = {}

        for i in range(0, len(nums)):
            n = target - nums[i]
            if n in h:
                return [h[n], i]
            h[nums[i]] = i
        
       